pages/1.11/gui/services/index.md

```diff
@@ -23,3 +23,22 @@ Click the service name to open the Instances panel, which provides CPU, memory,
 For services with a web interface, hover over the service name and click ![open service](/1.11/img/open-service.png) to view it.
 
 **Tip:** You can access the Mesos web interface at `<hostname>/mesos`.
+
+# Service Status
+
+DC/OS UI introduces the following concepts to illustrate the possible status of a Service at any point in time:
+
+## Running
+The Service is reported as running and no instances are reported as deploying or recovering.
```

Comment:
I think looks good, what do you think about `The Service is reported as running and no instances are deploying or recovering` though?